service/src/main/java/bio/terra/landingzone/service/landingzone/azure/model/LandingZoneSubnetResource.java

```diff
@@ -0,0 +1,98 @@
+package bio.terra.landingzone.service.landingzone.azure.model;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+public class LandingZoneSubnetResource {
```

Comment:
Could this class be a record?

Reply:
@sergiygetlin already pointed this one - working on it.